### PR TITLE
Fixes for vsnd, displays and consoles

### PR DESCRIPTION
**Pull request by aanisov**

### CODE REVIEW

File: recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend [hunk 1]
```diff
@@ -45,13 +45,7 @@ add_to_local_conf() {
...
     cd ${S}
 
-    # FIXME: although we do not install Xen we still run as Xen
-    # domain, so serial console, used for login shell must be the
-    # virtual one, e.g. hvc0
-    # FIXME: meta-virtualization layer will also update this
-    # when DISTRO_FEATURES has "xen" in it. But in our case it is not set,
-    # so still set up the console on our own
-    base_update_conf_value ${local_conf} SERIAL_CONSOLE "115200 hvc0"
+    base_update_conf_value ${local_conf} SERIAL_CONSOLE ""
```
Comment (andr2000) [inline recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend]: this needs a comment

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3-4x2g.cfg [hunk 2]
```diff
@@ -64,7 +64,7 @@ vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:768x1024' ]
 
 vkb = [ 'backend=DomD,backend-type=linux,multi-touch-width=1920,multi-touch-height=1080,multi-touch-num-contacts=10,id=T:1000' ]
 
-#vsnd = [[ 'card, backend=DomD, buffer-size=65536, short-name=VCard, long-name=Virtual sound card, sample-rates=48000, sample-formats=s16_le',
-#          'pcm, name=dev1', 'stream, id=alsa, type=P' ]]
+vsnd = [[ 'card, backend=DomD, buffer-size=32768, short-name=VCard, long-name=Virtual sound card, sample-rates=48000, sample-formats=s16_le',
```
[the remark this answers is not in the document]
Comment (aanisov) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-x-h3-4x2g.cfg]: OK, I'll keep this in mind.

File: recipes-domu/domu-image-fusion/domu-image-fusion.bbappend [hunk 3]
```diff
@@ -47,7 +47,11 @@ configure_versions() {
     # corresponding recipies.
     base_update_conf_value ${local_conf} BB_DANGLINGAPPENDS_WARNONLY "yes"
 
-    base_update_conf_value ${local_conf} SERIAL_CONSOLE "115200 hvc0"
+    # hvc0 is not a serial console, so is not processes properly by a modern
+    # start_getty script which is installed for sysvinit based systems.
+    # Instead a distro feature xen should be enabled in a configuration, so a
+    # direct call to getty with hvc0 is installed into inittab by meta-viltualization.
+    base_update_conf_value ${local_conf} SERIAL_CONSOLE ""
```
Comment (aanisov) [inline recipes-domu/domu-image-fusion/domu-image-fusion.bbappend]: Commented it.

**Comment by aanisov** [comment]
@iartemenko please merge

**Pull request merged**